driver/driver.feature

```diff
@@ -1391,7 +1657,6 @@ Feature: TypeDB Driver
       insert $dt 2023-05-01T00:00:00 Asia/Calcutta isa dt;
       """
     Then answer get row(0) get attribute(dt) get value is: 2023-05-01T00:00:00 Asia/Calcutta
-    Then answer get row(0) get attribute(dt) get value is: 2023-04-30T13:30:00 America/Chicago
```

Comment:
The comparison worked in Python and Rust's implementations of `Datetime`s, but the Java's implementation decided that different timezones mean different `ZonedDateTime`s (and I'm not really against it as they have a separate class for it). No need to check it there, the main thing is that we get what we inserted.